internal/isoeditor/rhcos_test.go

```diff
@@ -86,16 +86,11 @@ var _ = Context("with test files", func() {
 	})
 
 	Describe("CreateClusterMinimalISO", func() {
-		It("removes the workspace", func() {
+		It("cluster ISO created successfully", func() {
 			editor := editorForFile(isoFile, workDir)
 			proxyInfo := &ClusterProxyInfo{}
 			file, err := editor.CreateClusterMinimalISO("ignition", "", proxyInfo)
 			Expect(err).ToNot(HaveOccurred())
-
-			// Creating the template should remove the working directory
-			_, err = os.Stat(workDir)
-			Expect(os.IsNotExist(err)).To(BeTrue())
-
```

Comment:
We shouldn't leave this directory around even if we don't actually use the handler.
If this is failing then that means we're leaking a temp dir, right?

Reply:
This is the CreateClusterMinimalISO test, we don't create a template here. Returned the missing check in line 74.

Reply:
I know that, but are we still leaving the temp dir around in this case?

Reply:
The isoHandler's workdir is not used as we copy the iso to the isoeditor's workdir. 
The created cluster iso is removed by the caller, do you think we should remove the folder?
https://github.com/openshift/assisted-service/blob/0edec75edffd36b5402a151b6aa789fdf07bcce7/internal/bminventory/inventory.go#L1254

Reply:
> do you think we should remove the folder

If we don't who will?

Reply:
done